web/utils.go

```diff
@@ -34,6 +37,21 @@ func Userget(r *http.Request) (models.User, error) {
 	return user, nil
 }
 
+func encrypt(word string) (string, error) {
```

Comment:
This function is unneccesary. When storing password, you simply run  
`
hashedWord, err := bcrypt.GenerateFromPassword([]byte(word), bcrypt.DefaultCost)
`

and store the hashedPassword in the database as a binary. Not as a string. 

Then during login, you use
`
isMatch := bcrypt.CompareHashAndPassword(hashedWord, []byte(word))
`
The hasedWord would be the ninary password that was stored in the database.while word is the password text inserted by the user.

Bottom line is, there's no point for the two to be in the same function. Its just resource wastage. Dont forget encryption and decryption are not big O free operations.